data/json/items/migration.json

```diff
@@ -20,6 +20,12 @@
     "replace": "battery_car"
   },
   {
+    "id": "gold_small",
+    "type": "MIGRATION",
+    "replace": "gold",
+    "charges": 1
```

Comment:
It would probably be easier to drop the non-`small` gold/silver for now, instead migrating the bars into stacks of `small` gold/silver. Later the migration can be inverted when charge retention migration is supported.